src/views/MicroCredit/LoanRejected.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box,colors} from '@impact-market/ui';
+import Image from '../../libs/Prismic/components/Image';
+import RichText from '../../libs/Prismic/components/RichText';
+
+
+const LoanRejected = (props: any) => {
+    const { data } = props;
+    const {
+        rejectedLoanTitle,
+        rejectedLoanDescription,
+        rejectedLoanWhat,
+        rejectedLoanErr,
+        rejectedLoanRev,
+        rejectedLoanQuest,
+        rejectedLoanImage
+    } = data;
+
+    return (
+        <Box fLayout="start" flex fDirection={{ sm: 'row', xs: 'column' }}>
+            <Box flex fDirection={{ sm: 'row', xs: 'column' }}>
+                <Box
+                    style={{ flexBasis: '60%' }}
+                    center
+                    order={{ sm: 0, xs: 1 }}
+                >
+                    <Image {...rejectedLoanImage} radius={0.5} w="95%" mb={1} />
+                    <RichText content={rejectedLoanTitle} g800 medium />
+                    <RichText
+                        content={rejectedLoanDescription}
+                        small
+                        mt={0.5}
+                        g500
+                    />
+                </Box>
+
+                <Box
+                    ml={{ sm: '2rem', xs: 0 }}
+                    mb={{ sm: 0, xs: '1rem' }}
+                    fLayout="start"
+                    flex
+                    fDirection={{ sm: 'column', xs: 'row' }}
+                    order={{ sm: 1, xs: 0 }}
+                    style={{
+                        backgroundColor: colors.g100,
+                        borderRadius: '0.5rem',
+                        flexBasis: '65%',
+                        justifyContent: 'space-evenly',
+                        padding: '2rem'
+                    }}
```

Comment:
Here you can use the props `padding`, `backgroundColor` and `borderRadius` outside of style, like so:
````
<Box
   ...
   padding={2}
   backgroundColor={colors.g100}
   borderRadius={0.5}
   style={{
      flexBasis: '65%',
      justifyContent: 'space-evenly',
   }}
>